changelog/README.md

```diff
@@ -2,6 +2,7 @@
 
 ### 2.0.12 (in progress)
 
+- [bug] NPE in ControlConnection.updateLocationInfo (JAVA-656)
```

Comment:
Could you add your entry at the end of the list rather than the beginning?
It's just that it makes merges easier.